app/templates/views/guidance/features/security.html

```diff
@@ -9,79 +9,233 @@
+    <li><a class="govuk-link govuk-link--no-visited-state" href="#protecting-our-website-and-api">protecting our website and API</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="#email-security">email security</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="#security-classifications">security classifications</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="#staff">GOV.UK Notify staff</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="#suppliers">staff</a></li>
   </ul>
 
-  <h2 class="heading-medium" id="data">Data</h2>
-  <p class="govuk-body">On Notify, data is encrypted when:</p>
+  <h2 class="heading-medium" id="running-a-secure-service">Running a secure service</h2>
+  <p class="govuk-body">GOV.UK Notify:</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>it passes through the service</li>
-    <li>it’s stored on the service</li>
+    <li>follows the principles of the <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/service-standard">Service Standard</a></li>
+    <li>has been through a successful <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/service-assessments">live service assessment</a></li>
   </ul>
-  <p class="govuk-body">Any recipient data you upload is only held for 7 days.</p>
-  <p class="govuk-body">If you <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for("main.guidance_send_files_by_email") }}">send a file by email</a>, the file will be available for the recipient to download for up to 78 weeks (18 months).</p>
-  <p class="govuk-body">The Cabinet Office acts as data processor for Notify. Your organisation is the data controller.</p>
-  <h3 class="heading-small">Data Protection Act</h3>
-  <p class="govuk-body">Notify complies with data protection law. To make sure it stays compliant, there are regular legal reviews of the service’s:</p>
+  <p class="govuk-body">We regularly assess and review our security in line with:</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>privacy policy</li>
-    <li>terms of use</li>
-    <li>approach to data sharing</li>
+    <li>the <a class="govuk-link govuk-link--no-visited-state" href="https://www.security.gov.uk/guidance/govassure/">GovAssure approach</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/collection/caf/caf-principles-and-guidance">UK government cyber security standards</a></li>
+  </ul>
+<p class="govuk-body">We monitor the threat landscape and conduct regular <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/check-penetration-testing">CHECK penetration testing</a> so we can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>continue to improve our security</li>
+    </li>deal with common threats like Distributed Denial of Service (DDoS) attacks</li>
```

Comment:
This was causing the Concourse checks to fail

```suggestion
    <li>deal with common threats like Distributed Denial of Service (DDoS) attacks</li>
```